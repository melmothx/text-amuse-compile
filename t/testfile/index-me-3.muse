#title Index
#lang en

STARTHERE

<comment>
INDEX names: *Index*
First
Second
Third
Forth
Fifth
</comment>

* First Fifth

** Second

*** Third

        * * * * *

**** Forth

***** Fifth

        * * * * *

; First Second Third Forth Fifth

Here we go

ENDHERE
