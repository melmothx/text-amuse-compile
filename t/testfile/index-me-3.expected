





\part{First Fifth}
\index[names]{First}
\index[names]{Fifth}

\chapter{Second}
\index[names]{Second}

\section{Third}
\index[names]{Third}


\clearpage

\subsection{Forth}
\index[names]{Forth}

\subsubsection{Fifth}
\index[names]{Fifth}


\clearpage

\index[names]{First}
\index[names]{Second}
\index[names]{Third}
\index[names]{Forth}
\index[names]{Fifth}
% First Second Third Forth Fifth

Here we go


